### PR TITLE
[Doc] add 3.0.5 rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/release_notes/release-3.0.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+- BEs may crash when the AUTO_INCREMENT column is being updated. [#27199](https://github.com/StarRocks/starrocks/pull/27199)
+- Outer join and Anti join rewrite errors for materialized views. [#28028](https://github.com/StarRocks/starrocks/pull/28028)
+- Inaccurate estimation of average row size causes Primary Key partial updates to occupy excessively large memory. [#27485](https://github.com/StarRocks/starrocks/pull/27485)
+- Activating an inactive materialized view may cause a FE to crash. [#27959](https://github.com/StarRocks/starrocks/pull/27959)
+- Queries can not be rewritten to materialized views created based on external tables in the Hudi catalog. [#28023](https://github.com/StarRocks/starrocks/pull/28023)
+- You can still query the data of the Hive table even if you drop the Hive table and manually update the metadata cache. [#28223](https://github.com/StarRocks/starrocks/pull/28223)
+- Manually refreshing an asynchronous materialized view via a synchronous call results in multiple INSERT OVERWRITE records in the `information_schema.task_run` table. [#28060](https://github.com/StarRocks/starrocks/pull/28060)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:34]: name the table is incorrect, should be `task_runs`

File: docs/release_notes/release-3.0.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+- BEs may crash when the input values of the sub_bitmap function are not of the BITMAP type. [#27982](https://github.com/StarRocks/starrocks/pull/27982)
+- BEs may crash when the AUTO_INCREMENT column is being updated. [#27199](https://github.com/StarRocks/starrocks/pull/27199)
+- Outer join and Anti join rewrite errors for materialized views. [#28028](https://github.com/StarRocks/starrocks/pull/28028)
+- Inaccurate estimation of average row size causes Primary Key partial updates to occupy excessively large memory. [#27485](https://github.com/StarRocks/starrocks/pull/27485)
+- Activating an inactive materialized view may cause a FE to crash. [#27959](https://github.com/StarRocks/starrocks/pull/27959)
+- Queries can not be rewritten to materialized views created based on external tables in the Hudi catalog. [#28023](https://github.com/StarRocks/starrocks/pull/28023)
+- You can still query the data of the Hive table even if you drop the Hive table and manually update the metadata cache. [#28223](https://github.com/StarRocks/starrocks/pull/28223)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:33]: suggest: The data of a Hive table can still be queried even after the table is dropped and the metadata cache is manually updated.

File: docs/release_notes/release-3.0.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+- When the SELECT result set of INSERT INTO SELECT is empty, the output of SHOW LOAD shows the load job status is `CANCELED`. [#26913](https://github.com/StarRocks/starrocks/pull/26913)
+- BEs may crash when the input values of the sub_bitmap function are not of the BITMAP type. [#27982](https://github.com/StarRocks/starrocks/pull/27982)
+- BEs may crash when the AUTO_INCREMENT column is being updated. [#27199](https://github.com/StarRocks/starrocks/pull/27199)
+- Outer join and Anti join rewrite errors for materialized views. [#28028](https://github.com/StarRocks/starrocks/pull/28028)
+- Inaccurate estimation of average row size causes Primary Key partial updates to occupy excessively large memory. [#27485](https://github.com/StarRocks/starrocks/pull/27485)
+- Activating an inactive materialized view may cause a FE to crash. [#27959](https://github.com/StarRocks/starrocks/pull/27959)
+- Queries can not be rewritten to materialized views created based on external tables in the Hudi catalog. [#28023](https://github.com/StarRocks/starrocks/pull/28023)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:32]: in a Hudi catalog

File: docs/release_notes/release-3.0.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+
+## Bug Fixes
+
+Fixed the following issues:
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#27779](https://github.com/StarRocks/starrocks/pull/27779)
+- The query result is not stable if the sort key is modified for a partitioned table. [#27850](https://github.com/StarRocks/starrocks/pull/27850)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored. [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- The number of buckets is automatically inferred as 0 if the number of buckets is not specified at the creation of a Colocation table, which causes failure of adding new partitions. [#27086](https://github.com/StarRocks/starrocks/pull/27086)
+- When the SELECT result set of INSERT INTO SELECT is empty, the output of SHOW LOAD shows the load job status is `CANCELED`. [#26913](https://github.com/StarRocks/starrocks/pull/26913)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:26]: suggest: the load job status returned by SHOW LOAD is`CANCELED`.

File: docs/release_notes/release-3.0.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+- The partitioning field is case-insensitive when expression partitioning is used at table creation and [INSERT OVERWRITE is used to overwrite data in a specific partition](../table_design/expression_partitioning.md#load-data-into-partitions). [#28309](https://github.com/StarRocks/starrocks/pull/28309)
+
+## Bug Fixes
+
+Fixed the following issues:
+
+- Incorrect table-level scan statistics in FE cause inaccurate metrics for table queries and loading. [#27779](https://github.com/StarRocks/starrocks/pull/27779)
+- The query result is not stable if the sort key is modified for a partitioned table. [#27850](https://github.com/StarRocks/starrocks/pull/27850)
+- The version number for a tablet is inconsistent between the BE and FE after data is restored. [#26518](https://github.com/StarRocks/starrocks/pull/26518/files)
+- The number of buckets is automatically inferred as 0 if the number of buckets is not specified at the creation of a Colocation table, which causes failure of adding new partitions. [#27086](https://github.com/StarRocks/starrocks/pull/27086)
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:25]: The number of buckets is inferred as 0 if users did not specify the bucket number when they create a Colocation table, which causes failures in adding new partitions.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**